models/build.js

```diff
@@ -66,6 +66,11 @@ const MODEL = {
         .description('SHA this project was built on')
         .example('ccc49349d3cffbd12ea9e3d41521480b4aa5de5f'),
 
+    subscribedConfigSha: Joi
```

Comment:
Do we need to add migration file to add this column on builds?
https://github.com/screwdriver-cd/data-schema/tree/master/migrations